code/game/objects/items/clown_items.dm

```diff
@@ -150,10 +150,11 @@
 	throw_speed = 3
 	throw_range = 7
 	attack_verb = list("HONKED")
+	datum_outputs = list(/datum/outputs/bikehorn)

```

Comment:
You do not need to initialize a list in this object since what actually uses do is squeak datum